config.go

```diff
@@ -227,7 +227,7 @@ type config struct {
 	NoSeedBackup bool `long:"noseedbackup" description:"If true, NO SEED WILL BE EXPOSED AND THE WALLET WILL BE ENCRYPTED USING THE DEFAULT PASSPHRASE -- EVER. THIS FLAG IS ONLY FOR TESTING AND IS BEING DEPRECATED."`
 
 	TrickleDelay        int           `long:"trickledelay" description:"Time in milliseconds between each release of announcements to the network"`
-	InactiveChanTimeout time.Duration `long:"inactivechantimeout" description:"If a channel has been inactive for the set time, send a ChannelUpdate disabling it."`
+	InactiveChanTimeout time.Duration `long:"inactivechantimeout" description:"If a channel has been inactive for the set time, send a ChannelUpdate disabling it. If the channel becomes active again, send an enabling ChannelUpdate after the same timeout."`
```

Comment:
I think it'd be useful to have these be distinctly configurable. The naming is somewhat confusing since the config option is overloaded

Reply:
would you say it is sufficient to rename it? Making them distinct will complicate the logic a bit, and also seems they should be symmetric.

Reply:
hmm my thinking was that they would be asymmetric, or at least that the ability to configure them as such would be useful even if the defaults were the same

Reply:
this is also somewhat of a breaking change from the user's perspective. prior to this change, their channels will be reenabled immediately. after applying this change, their channels won't be routable for 20 mins. this behavior happens w/o making any changes to the config because we're overloading the param

Reply:
It should help with the health of the network, and in the future nodes might get punished (by getting used less for routing) for not disabling unreliable channels.

Reply:
> It should help with the health of the network

I agree, which to me indicates that the defaults should be symmetric. though i still don't see a reason for not giving users the option to choose asymmetric timeouts if they want, esp if the code paths for enable/disable will be distinct anyway.

misnaming/overloading parameters has led to a great deal of confusion previously, imo we should try keep argument names as self-descriptive as possible, since most users will never read `lnd -h`

Reply:
Agree on changing the parameter name to something more descriptive. 

But I'm reluctant to making it possible to set them asymmetrically, as we want to discourage that (I think?), and also clutters up the config.